pages/faq.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">FAQ with Professor Julian Goodare</h1>
+    <p class="mb-6 text-justify">
+      University of Edinburgh students, Ruby Imrie and Ellie Whitehead, sat down
+      with Professor Julian Goodare, lecturer in Scottish History, to ask him
```

Comment:
We should link to the ed.ac.uk page about Julian again here. Thanks Ruby

Reply:
done thanks

Reply:
thank you. Again, the styling is missing but I assume this will be fixed once a merge takes place yes?

Reply:
ahh I've just seen your comment. Will mark as resolved.